beta/src/pages/learn/index.md

```diff
@@ -4,716 +4,486 @@ title: Quick Start
 
 <Intro>
 
-Welcome to the React documentation! Here is an overview of what you can find on this site.
+Welcome to the React documentation! This page will give you an introduction to the 80% of React concepts that you will use on a daily basis.
 
 </Intro>
 
 <YouWillLearn>
 
-* [How to set up React](/learn/installation)
-* [How to think about building user interfaces with React](/learn/thinking-in-react)
-* [How to get something rendering on the screen](/learn/describing-the-ui)
-* [How to make your user interface respond to interactions](/learn/adding-interactivity)
-* [How to keep your logic maintainable as your app grows](/learn/managing-state)
-* [How to step outside React when necessary](/learn/escape-hatches)
+- How to define components
+- How to add markup and styles
+- How to display data
+- How to render conditions and lists
+- How to respond to events and update the screen
+- How to share data between components
 
 </YouWillLearn>
 
```

Comment:
It kind of feels like a brief one or two sentence overview is missing here, like how all these pieces (components, JSX, state) all fit together.

Reply:
I don't disagree but not sure what to write. I'll get this in as is and revisit when we flesh out related pages and think about landing etc.